### PR TITLE
#673: Polish scroll arrows (color scroll arrow)

### PR DESCRIPTION
**Pull request by BeeMargarida**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-white/issues/673 (A PR in `ripe-white` will be made with the enabling of these scroll arrows) |
| Dependencies | -- |
| Problems | pickers-colors-not-disappearing-problemcolor-pickers-scroll-arrow-problem|
| Decisions | The problems were related to the color list div size not being updated when the `updateScrollFlags` function was called. The solution was adding the `after-enter` hook to the `transition-group` and update the scroll flags when the last element on the list enters the animation. (Another PR will be made with fixes to the scrolling experience). |
| Animated GIF | pickers-working-wellpickers-working-well-desktop |

### CODE REVIEW

File: vue/components/organisms/pickers/pickers.vue [hunk 1]
```diff
@@ -13,7 +13,7 @@
                 class="button-scroll button-scroll-left button-scroll-parts button-scroll-parts-left"
                 v-if="enableScrollParts"
                 v-show="scrollParts"
-                v-on:click="slideLeftParts"
+                v-on:click.stop="slideLeftParts"
```
Comment (gcandal) [inline vue/components/organisms/pickers/pickers.vue:16]: Why is this needed?

Comment (BeeMargarida) [reply vue/components/organisms/pickers/pickers.vue:16]: So that this behaviour (gif) doesn't happen. I think it is a bit weird that by clicking an arrow to scroll horizontally, the panel opens. It should only open when clicking a part.
pickers-stop

**Review by gcandal — APPROVED** [review]

**Review by joamag — CHANGES_REQUESTED** [review]

File: vue/components/organisms/pickers/pickers.vue [hunk 2]
```diff
@@ -396,6 +400,7 @@ body.mobile .button-scroll-materials {
     left: 0px;
 }
 
+body.tablet .button-scroll-colors,
```
Comment (joamag) [inline vue/components/organisms/pickers/pickers.vue:403]: are we sure about this? I believe the pickers in tablet are pretty much the same as in desktop? @gcandal can you confirm?

**Review by joamag — CHANGES_REQUESTED** [review]

**Comment by joamag** [comment]
@gcandal @BeeMargarida we'll also need to fix the merge conflicts

File: vue/components/organisms/pickers/pickers.vue (hunk 2 above)
Comment (BeeMargarida) [reply vue/components/organisms/pickers/pickers.vue:403]: It resulted from this difference when the `white-pickers` style is removed:<br>
Without change:
Screenshot_2020-12-07 RIPE White(1)
With change:
Screenshot_2020-12-07 RIPE White

**Comment by BeeMargarida** [comment]
@gcandal @joamag Pickers not showing when collapsed:
Screenshot_2020-12-07 RIPE White(3)
It requires to pass `pickersCollapsed` value from `main-content` to `pickers`

**Comment by joamag** [comment]
> @gcandal @joamag Pickers not showing when collapsed:
> Screenshot_2020-12-07 RIPE White(3)
> It requires to pass `pickersCollapsed` value from `main-content` to `pickers` or apply a `::v-deep` in `white-pickers`.

The same thing for the horizontal scrol

**Comment by BeeMargarida** [comment]
@joamag I think this branch is done, the changes to remove the horizontal scroll and scroll arrows are being done in `ripe-white` (I will update the PR in draft), due to the fact that pickers does not know it is collapsed. 
Screenshot_2020-12-07 RIPE White(4)

File: vue/components/organisms/pickers/pickers.vue (hunk 2 above)
Comment (gcandal) [reply vue/components/organisms/pickers/pickers.vue:403]: @joamag there's a variation in the pickers: in tablet there is no close button in the pickers (`X`)

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Comment by joamag** [comment]
@gcandal please try to fix these conflicts... in case this PR is still relevant

**Review by gcandal — APPROVED** [review]

**Review by joamag — APPROVED** [review]
